arangodb-net-standard/GraphApi/PostVertexResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Net;
+
+namespace ArangoDBNetStandard.GraphApi
+{
+    public class PostVertexResponse
+    {
+        public PostVertexResult New { get; set; }
```

Comment:
`New` should be of type `T` according to the documentation:

> The complete newly written vertex document. Includes all written attributes in the request body and all internal attributes generated by ArangoDB. Will only be present if returnNew is true.

Let's also document the Vertex and New properties here. We can also include a property ref?
Example:

`The complete newly written vertex document. Includes all written attributes in the request body and all internal attributes generated by ArangoDB. Will only be present if <see cref="PostVertexQuery.ReturnNew"/> is true.`

You can find an existing example with `PostDocumentResponse`.